lib/portlayer/portlayer.go

```diff
@@ -83,5 +84,46 @@ func Init(ctx context.Context, sess *session.Session) error {
 		return err
 	}
 
+	unbindSerialPorts(sess)
```

Comment:
You may want to add a comment hear as to why we are calling this here. I know there is a comment on the function definition. But the init code should also be as unambiguous as possible imo :)